Sources/Apodini/Semantic Model Builder/REST/RESTInterfaceExporter.swift

```diff
@@ -165,7 +164,16 @@ class RESTInterfaceExporter: InterfaceExporter {
                 return nil
             }
 
-            return try request.content.decode(Type.self, using: JSONDecoder())
+            #warning("""
+                     A Handler could define multiple .content Parameters. In such a case the REST exporter would
+                     need to decode the content via a struct containing those .content parameters as properties.
+                     This is currently unsupported.
+                """)
+            do {
+                return try request.content.decode(Type.self, using: JSONDecoder())
```

Comment:
This is due to having two Parameter properties in the `Update` handler of which one is nil. Currently there is no support for requests contain more than one body parameter. If the request contains something like
```
let parameters: [String: TypeContainer] = [
            "name": TypeContainer(with: "FooBird")
]
```
It would try and decode it as a `Bird` which would fail and throw a `DecodingError`. But as this can also be decoded as `Parameter<[String: TypeContainer]>, it should fail silently

Reply:
Ah, makes sense, thanks for the explanation!